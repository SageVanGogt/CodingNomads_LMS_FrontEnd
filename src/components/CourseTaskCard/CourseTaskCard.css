.CourseTaskCard {
  width: 300px;
  border: 1px dashed black;
  margin: 5px auto; }

.CourseTaskCard:hover {
  cursor: pointer; }

.CourseTaskCard h4 {
  display: inline-block; }

.CourseTaskCard img {
  float: right;
  padding: 3px 3px 0px 0px;
  cursor: initial;
  width: 10px;
  height: 10px; }
