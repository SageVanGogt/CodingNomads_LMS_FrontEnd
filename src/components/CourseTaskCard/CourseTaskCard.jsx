import React from 'react';
import './CourseTaskCard.css';

export const CourseTaskCard = (props) => {
  const {id, name, description} = props;

  return (
    <div className='CourseTaskCard' draggable onDragStart={(e) => console.log(e)}>
      <h4 title={description}>{name}</h4>
    </div>
  );
};