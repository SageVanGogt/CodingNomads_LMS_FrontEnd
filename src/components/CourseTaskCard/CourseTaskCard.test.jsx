import React from 'react';
import { shallow } from 'enzyme';
import { CourseTaskCard } from './CourseTaskCard';

describe('CourseTaskCard', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<CourseTaskCard />)

    expect(wrapper).toMatchSnapshot();
  });
});
