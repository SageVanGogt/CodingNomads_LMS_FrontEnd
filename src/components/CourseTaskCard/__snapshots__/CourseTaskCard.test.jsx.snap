// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourseTaskCard matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h4
          display="inline-block"
          title={undefined}
/>,
        <img
          onClick={[Function]}
          src="/button.png"
/>,
      ],
      "className": "CourseTaskCard",
      "draggable": true,
      "onDragStart": [Function],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "display": "inline-block",
          "title": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "onClick": [Function],
          "src": "/button.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4
            display="inline-block"
            title={undefined}
/>,
          <img
            onClick={[Function]}
            src="/button.png"
/>,
        ],
        "className": "CourseTaskCard",
        "draggable": true,
        "onDragStart": [Function],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "display": "inline-block",
            "title": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "onClick": [Function],
            "src": "/button.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
